src/sql/helpers.js

```diff
@@ -64,26 +64,44 @@ const jsonbObjectAgg = (aggregationColumnKey, nestedFields, aggregationJsonKey,
  * For this example:
  * - rootFields: ['id', 'name'],
  * - nestedFields: ['pipeline_arn', 'hash]
- * - aggregationColumnKey: 'pipeline_type'
- * - aggregationJsonKey: 'pipelines'
+ * - collapseKey: 'pipeline_type'
+ * - collapseKeyNewName: 'pipelines'
  *
  * Note it also camelcases all the keys returned
  */
-const aggregateIntoJson = (
+const collapseKeysIntoObject = (
   originalQuery,
```

Comment:
why don't you rename `originalQuery` and `mainQuery` names used in here and in `model/experiment.js` to all be called `query` or `mainQuery`? If they match names, the code will be clearers that they are the same thing. `query` can be a good name, because they are all essentially the query.